src/buffer/out/textBuffer.cpp

```diff
@@ -1140,12 +1136,12 @@ const COORD TextBuffer::GetWordStart(const COORD target, const std::wstring_view
     {
         // GH#7664: Treat EndExclusive as EndInclusive so
         // that it actually points to a space in the buffer
-        copy = { bufferSize.RightInclusive(), bufferSize.BottomInclusive() };
+        copy = bufferSize.BottomRightInclusive();
     }
-    else if (bufferSize.CompareInBounds(target, limit.to_win32_coord(), true) >= 0)
+    else if (bufferSize.CompareInBounds(target, limit, true) >= 0)
     {
         // if at/past the limit --> clamp to limit
-        copy = limitOptional->to_win32_coord();
+        copy = *limitOptional;

```

Comment:
I'll fix this one right away via:
```cpp
limitOptional.value_or(bufferSize.BottomRightInclusive())
```
Because that matches the `limit` variable if `limitOptional` isn't set.